src/card.css

```diff
@@ -0,0 +1,105 @@
+.card {
+  font-family: Roboto, sans-serif;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 28px 16px;
```

Comment:
The bottom padding should be 16px

![image](https://user-images.githubusercontent.com/63306833/173759036-7b544fa7-8eb8-4934-b55b-a500dc71de16.png)